panic: runtime error: integer divide by zero

Hello,

I sometimes see the bug below.

But, There is no PoC for reproduction.

Thanks.


```
panic: runtime error: integer divide by zero

goroutine 72 [running]:
github.com/OWASP/Amass/amass.(*NameService).processRequests(0xc0000a2960)
        /build/amass/parts/amass/go/src/github.com/OWASP/Amass/amass/namesrv.go:89 +0x793
created by github.com/OWASP/Amass/amass.(*NameService).OnStart
        /build/amass/parts/amass/go/src/github.com/OWASP/Amass/amass/namesrv.go:52 +0x63
```

--- a/amass/namesrv.go
+++ b/amass/namesrv.go
@@ -86,7 +86,9 @@ func (ns *NameService) processRequests() {
 			for _, s := range perSec {
 				total += s
 			}
-			ns.Enum().Log.Printf("Average DNS names processed: %d/sec", total/num)
+			if num > 0 {
+				ns.Enum().Log.Printf("Average DNS names processed: %d/sec", total/num)
+			}
 			perSec = []int{}
 		case req := <-ns.RequestChan():
 			go ns.performRequest(req)

